packages/kit/src/core/dev/index.js

```diff
@@ -187,7 +187,8 @@ class Watcher extends EventEmitter {
 							},
 							hooks: {
 								getSession: hooks.getSession || (() => ({})),
-								handle: hooks.handle || (({ request, render }) => render(request))
+								handle: hooks.handle || (({ request, render }) => render(request)),
+								serverFetch: hooks.serverFetch
```

Comment:
should this also provide a default implementation like `serverFetch: hooks.serverFetch || fetch`? then you can just call it directly without having to check if it's defined

Reply:
hmm maybe it's a good idea, I can check it.

Reply:
Thanks! During these changes, I found and fix the serious issue.